### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by arthurvandijk**
Initial set up for the main rubicon branch containing repo guidelines and templates.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,27 @@
...
+### Default branch
+Master is the default branch for the repository
+
+### Branch protection rules
+The following protection rules must be enabled en configured
+* Require pull request reviews before merging. *At least 1 approving review is required*
+* Require review from Code Owners
+* Require status checks to pass before merging
+  * Require branches to be up to date before merging
+* Restrict who can push to mathcing branches
```
Comment (jpaarhuis) [inline README.md]: typo: mathcing

File: README.md [hunk 2]
```diff
@@ -1,2 +1,27 @@
...
+## Repository guidelines
+To ensure a way of working and a level of code quality newly created repositories must have at least the following settings;
+
+### Default branch
+Master is the default branch for the repository
+
+### Branch protection rules
+The following protection rules must be enabled en configured
+* Require pull request reviews before merging. *At least 1 approving review is required*
+* Require review from Code Owners
```
Comment (jpaarhuis) [inline README.md]: Combine these two bullet points, Almost seems as it require reviews from all Code Owners, while it only require one review from any of the Code Owners.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,27 @@
...
+* Require status checks to pass before merging
+  * Require branches to be up to date before merging
+* Restrict who can push to mathcing branches
+
+## Issue templates
+When your repository will be used by users outside the organisation its useful to set up issue templates.
+For now you can use the default *Bug* and *Feature* template given by github.
+
+## Action templates
+In the [workflow-template](https://github.com/rubicon-nl/rubicon-nl/tree/master/workflow-templates) folder you can find action templates for the following project types
```
Comment (jpaarhuis) [inline README.md]: Nice work on the templates! Are you also planning on (maybe optionally) include testing steps?

**Review by jpaarhuis — CHANGES_REQUESTED** [review]

**Comment by jpaarhuis** [comment]
Volgens mij moet hier nog wat gebeuren.

File: README.md (hunk 3 above)
Comment (arthurvandijk) [reply README.md]: Adding (optional) testing steps is planned and will be added in the next release. 
The dotnet core and Angular templates will be updated with default (optional) testing steps

**Review by jpaarhuis — APPROVED** [review]

**Pull request merged**